lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -497,5 +519,7 @@ def _get(
         query = self._build_sql_query(owner, json.get("access"), query)
         if json.get("keysummary"):
             return jsonify(self.keyspace(query))
+        elif json.get("daterange"):
+            return jsonify(self.daterange(request, json, query))
         else:
             return jsonify(self.datasets(request, json, query))
```

Comment:
Do we "like" having `"keysummary"` quietly override `"daterange"`?

The obvious alternatives being:
- since we return a JSON object anyway, we could return both the key summary and the date range, if they are both requested; or,
- we could return an error if both `"keysummary"` and `"daterange"` are specified.

The documentation doesn't describe what is supposed to happen if both are specified.  (BTW, in the documentation, the `keysummary` section is not in alphabetical order, like the rest of the keys.)

Reply:
I wouldn't so much say that I "like" it as that I didn't see a reason to care enough to worry about it. For one thing, I'd really need to demote the keysummary output to a sub-object, to combine them, and while I spent whole nanoseconds carefully weighing this idea during coding, I decided to be lazy and not mess with more tests. E.g., something like `{"from": "<date>", "to": "<date>", "keys": {"dataset": {...}}}`

Then again, maybe that's not such a bad idea since the UI could make a single "setup" call to get the key namespace and date range on page load instead of needing to make separate calls...

Reply:
So, I'll allow them to be combined. Why not?